torchx/specs/named_resources_aws.py

```diff
@@ -255,7 +256,11 @@ def aws_g5_48xlarge() -> Resource:
 
 def aws_trn1_2xlarge() -> Resource:
     return Resource(
-        cpu=8, gpu=0, memMB=32 * GiB, capabilities={K8S_ITYPE: "trn1.2xlarge"}
+        cpu=8,
+        gpu=0,
+        memMB=32 * GiB,
+        capabilities={K8S_ITYPE: "trn1.2xlarge"},
+        devices={NEURON_DEVICE: 1},
```

Comment:
How were these being used before without these device mounts? Was there a init script/host config that handled this?

Reply:
There is a optional package aws-neuronx-oci-hook which allows for setting “AWS_NEURON_VISIBLE_DEVICES”
https://awsdocs-neuron.readthedocs-hosted.com/en/latest/containers/tutorials/build-run-neuron-container.html#container-devices

Which is similar to the nvidia container setting CUDA_VISIBLE_DEVICES
And for kubernetes, there is a plugin which uses aws.amazon.com/neurondevice